packages/react/src/vm-extension/__tests__/vm-extension-details-panel.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import { render } from "@testing-library/react";
+import { VmExtensionDetailsPanel } from "../vm-extension-details-panel";
+import { succeededExtItem } from "../../test-util/mock-vm-ext-items";
+import * as React from "react";
+import { initMockBrowserEnvironment } from "@azure/bonito-ui";
+import { runAxe } from "@azure/bonito-ui/lib/test-util/a11y";
+import { translate } from "@azure/bonito-core";
+
+describe("VmExtensionDetailsPanel", () => {
+    beforeEach(() => initMockBrowserEnvironment());
+
+    it("should render if isOpen", async () => {
+        const onDismiss = jest.fn();
+        const { container, getByText } = render(
+            <VmExtensionDetailsPanel
+                isOpen={true}
+                vme={succeededExtItem}
+                onDismiss={onDismiss}
+            />
+        );
+        getByText(translate("lib.react.vmExtension.extensionProperties"));
+        expect(await runAxe(container)).toHaveNoViolations();
+    });
+
+    it("should not render if not isOpen", async () => {
+        const onDismiss = jest.fn();
+        const { getByText } = render(
+            <VmExtensionDetailsPanel
+                isOpen={false}
+                vme={succeededExtItem}
+                onDismiss={onDismiss}
+            />
+        );
+        expect(() =>
+            getByText(translate("lib.react.vmExtension.extensionProperties"))
+        ).toThrow();
```

Comment:
Can we assert positively that there's an empty or hidden element or something like that instead of just checking that the text doesn't exist?

Reply:
This one is hard. When the panel opens, it actually adds a <div> under <body> and removes it when closes.

Reply:
Happy to follow up on this, but this isn't a total blocker. I'll just resolve and approve since this PR has been out for so long.